tools/snpsift_filter_builder/.shed.yml

```diff
@@ -0,0 +1,14 @@
+# .shed.yml supporting automatic pushes.
+owner: artbio
+name: snpsift_filter_builder
+description:  snpsift_filter_builder creates the filter line parameter to filter variants with the SnpSift tool
```

Comment:
snpsift_filter_builder creates the filter line parameter `for filtering variants` with the SnpSift tool